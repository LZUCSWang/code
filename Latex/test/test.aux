\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{nty/global//global/global}
\abx@aux@cite{0}{4358754}
\abx@aux@segm{0}{0}{4358754}
\providecommand \oddpage@label [2]{}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\@writefile{toc}{\beamer@sectionintoc {1}{绪论}{3}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{2}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{2}}}
\@writefile{nav}{\headcommand {\sectionentry {1}{绪论}{3}{绪论}{0}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{4/4}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {4}{4}}}
\newlabel{fig:my_label}{{1}{5}{主要研究内容}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{5}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {5}{5}}}
\@writefile{toc}{\beamer@sectionintoc {2}{问题建模}{6}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {3}{5}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {3}{5}}}
\@writefile{nav}{\headcommand {\sectionentry {2}{问题建模}{6}{问题建模}{0}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{1}{多无人机协同的草原修复问题}{6}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {6}{5}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{1}{6}{多无人机协同的草原修复问题}}}
\newlabel{fig:my_label}{{2}{6}{多无人机修复退化区域实例}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{6}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{1}{6/6}{多无人机协同的草原修复问题}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {6}{6}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{2}{多无人机协同的草原修复面积最大化模型}{7}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {6}{6}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{2}{7}{多无人机协同的草原修复面积最大化模型}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{1}{7/7}{多无人机协同的草原修复面积最大化模型}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {7}{7}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{2}{8/8}{多无人机协同的草原修复面积最大化模型}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {8}{8}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{3}{数学模型}{9}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {7}{8}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{3}{9}{数学模型}}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{1}{9/9}{数学模型}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{9}}}
\@writefile{toc}{\beamer@sectionintoc {3}{求解方法}{10}{0}{3}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {6}{9}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {9}{9}}}
\@writefile{nav}{\headcommand {\sectionentry {3}{求解方法}{10}{求解方法}{0}}}
\@writefile{toc}{\beamer@subsectionintoc {3}{1}{基于强化学习的最优路径规划}{10}{0}{3}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {10}{9}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{3}{1}{10}{基于强化学习的最优路径规划}}}
\@writefile{nav}{\headcommand {\slideentry {3}{1}{1}{10/10}{基于强化学习的最优路径规划}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {10}{10}}}
\newlabel{fig:my_label}{{3}{11}{网络结构图}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{11}}
\@writefile{nav}{\headcommand {\slideentry {3}{1}{2}{11/11}{基于强化学习的最优路径规划}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {11}{11}}}
\@writefile{nav}{\headcommand {\slideentry {3}{1}{3}{12/12}{基于强化学习的最优路径规划}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {12}{12}}}
\@writefile{toc}{\beamer@subsectionintoc {3}{2}{多无人机协同调度算法}{13}{0}{3}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {10}{12}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{3}{2}{13}{多无人机协同调度算法}}}
\newlabel{fig:my_label}{{4}{13}{多无人机修复退化区域实例}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{13}}
\@writefile{nav}{\headcommand {\slideentry {3}{2}{1}{13/13}{多无人机协同调度算法}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {13}{13}}}
\@writefile{toc}{\beamer@sectionintoc {4}{仿真结果}{14}{0}{4}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {10}{13}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {13}{13}}}
\@writefile{nav}{\headcommand {\sectionentry {4}{仿真结果}{14}{仿真结果}{0}}}
\newlabel{fig:my_label}{{5}{14}{仿真参数设置}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{14}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{1}{14/14}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {14}{14}}}
\newlabel{fig:my_label}{{6}{15}{模拟训练过程}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{15}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{2}{15/15}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {15}{15}}}
\newlabel{fig:my_label}{{7}{16}{无人机修复详情}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{16}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{3}{16/16}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {16}{16}}}
\newlabel{fig:my_label}{{8}{17}{无人机修复轨迹}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {fig:my_label}{17}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{4}{17/17}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {17}{17}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{5}{18/18}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {18}{18}}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{18}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {14}{18}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {14}{18}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {18}}}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {17}}}
\abx@aux@read@bbl@mdfivesum{nohash}
\abx@aux@read@bblrerun
\gdef \@abspage@last{18}
