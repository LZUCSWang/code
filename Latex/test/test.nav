\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\headcommand {\beamer@framepages {1}{1}}
\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}
\headcommand {\beamer@framepages {2}{2}}
\headcommand {\beamer@sectionpages {1}{2}}
\headcommand {\beamer@subsectionpages {1}{2}}
\headcommand {\sectionentry {1}{绪论}{3}{绪论}{0}}
\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}
\headcommand {\beamer@framepages {3}{3}}
\headcommand {\slideentry {1}{0}{2}{4/4}{}{0}}
\headcommand {\beamer@framepages {4}{4}}
\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}
\headcommand {\beamer@framepages {5}{5}}
\headcommand {\beamer@sectionpages {3}{5}}
\headcommand {\beamer@subsectionpages {3}{5}}
\headcommand {\sectionentry {2}{问题建模}{6}{问题建模}{0}}
\headcommand {\beamer@subsectionpages {6}{5}}
\headcommand {\beamer@subsectionentry {0}{2}{1}{6}{多无人机协同的草原修复问题}}
\headcommand {\slideentry {2}{1}{1}{6/6}{多无人机协同的草原修复问题}{0}}
\headcommand {\beamer@framepages {6}{6}}
\headcommand {\beamer@subsectionpages {6}{6}}
\headcommand {\beamer@subsectionentry {0}{2}{2}{7}{多无人机协同的草原修复面积最大化模型}}
\headcommand {\slideentry {2}{2}{1}{7/7}{多无人机协同的草原修复面积最大化模型}{0}}
\headcommand {\beamer@framepages {7}{7}}
\headcommand {\slideentry {2}{2}{2}{8/8}{多无人机协同的草原修复面积最大化模型}{0}}
\headcommand {\beamer@framepages {8}{8}}
\headcommand {\beamer@subsectionpages {7}{8}}
\headcommand {\beamer@subsectionentry {0}{2}{3}{9}{数学模型}}
\headcommand {\slideentry {2}{3}{1}{9/9}{数学模型}{0}}
\headcommand {\beamer@framepages {9}{9}}
\headcommand {\beamer@sectionpages {6}{9}}
\headcommand {\beamer@subsectionpages {9}{9}}
\headcommand {\sectionentry {3}{求解方法}{10}{求解方法}{0}}
\headcommand {\beamer@subsectionpages {10}{9}}
\headcommand {\beamer@subsectionentry {0}{3}{1}{10}{基于强化学习的最优路径规划}}
\headcommand {\slideentry {3}{1}{1}{10/10}{基于强化学习的最优路径规划}{0}}
\headcommand {\beamer@framepages {10}{10}}
\headcommand {\slideentry {3}{1}{2}{11/11}{基于强化学习的最优路径规划}{0}}
\headcommand {\beamer@framepages {11}{11}}
\headcommand {\slideentry {3}{1}{3}{12/12}{基于强化学习的最优路径规划}{0}}
\headcommand {\beamer@framepages {12}{12}}
\headcommand {\beamer@subsectionpages {10}{12}}
\headcommand {\beamer@subsectionentry {0}{3}{2}{13}{多无人机协同调度算法}}
\headcommand {\slideentry {3}{2}{1}{13/13}{多无人机协同调度算法}{0}}
\headcommand {\beamer@framepages {13}{13}}
\headcommand {\beamer@sectionpages {10}{13}}
\headcommand {\beamer@subsectionpages {13}{13}}
\headcommand {\sectionentry {4}{仿真结果}{14}{仿真结果}{0}}
\headcommand {\slideentry {4}{0}{1}{14/14}{}{0}}
\headcommand {\beamer@framepages {14}{14}}
\headcommand {\slideentry {4}{0}{2}{15/15}{}{0}}
\headcommand {\beamer@framepages {15}{15}}
\headcommand {\slideentry {4}{0}{3}{16/16}{}{0}}
\headcommand {\beamer@framepages {16}{16}}
\headcommand {\slideentry {4}{0}{4}{17/17}{}{0}}
\headcommand {\beamer@framepages {17}{17}}
\headcommand {\slideentry {4}{0}{5}{18/18}{}{0}}
\headcommand {\beamer@framepages {18}{18}}
\headcommand {\beamer@partpages {1}{18}}
\headcommand {\beamer@subsectionpages {14}{18}}
\headcommand {\beamer@sectionpages {14}{18}}
\headcommand {\beamer@documentpages {18}}
\headcommand {\gdef \inserttotalframenumber {17}}
