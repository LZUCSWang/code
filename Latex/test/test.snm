\beamer@slide {fig:my_label}{5}
\beamer@slide {fig:my_label}{6}
\beamer@slide {fig:my_label}{11}
\beamer@slide {fig:my_label}{13}
\beamer@slide {fig:my_label}{14}
\beamer@slide {fig:my_label}{15}
\beamer@slide {fig:my_label}{16}
\beamer@slide {fig:my_label}{17}
