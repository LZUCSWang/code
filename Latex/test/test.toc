\beamer@sectionintoc {1}{绪论}{3}{0}{1}
\beamer@sectionintoc {2}{问题建模}{6}{0}{2}
\beamer@subsectionintoc {2}{1}{多无人机协同的草原修复问题}{6}{0}{2}
\beamer@subsectionintoc {2}{2}{多无人机协同的草原修复面积最大化模型}{7}{0}{2}
\beamer@subsectionintoc {2}{3}{数学模型}{9}{0}{2}
\beamer@sectionintoc {3}{求解方法}{10}{0}{3}
\beamer@subsectionintoc {3}{1}{基于强化学习的最优路径规划}{10}{0}{3}
\beamer@subsectionintoc {3}{2}{多无人机协同调度算法}{13}{0}{3}
\beamer@sectionintoc {4}{仿真结果}{14}{0}{4}
