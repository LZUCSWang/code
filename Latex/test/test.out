\BOOKMARK [2][]{Outline0.1}{\376\377\176\352\213\272}{}% 1
\BOOKMARK [2][]{Outline0.2}{\376\377\225\356\230\230\136\372\152\041}{}% 2
\BOOKMARK [3][]{Outline0.2.1.6}{\376\377\131\032\145\340\116\272\147\072\123\117\124\014\166\204\203\111\123\237\117\356\131\015\225\356\230\230}{Outline0.2}% 3
\BOOKMARK [3][]{Outline0.2.2.7}{\376\377\131\032\145\340\116\272\147\072\123\117\124\014\166\204\203\111\123\237\117\356\131\015\227\142\171\357\147\000\131\047\123\026\152\041\127\213}{Outline0.2}% 4
\BOOKMARK [3][]{Outline0.2.3.9}{\376\377\145\160\133\146\152\041\127\213}{Outline0.2}% 5
\BOOKMARK [2][]{Outline0.3}{\376\377\154\102\211\343\145\271\154\325}{}% 6
\BOOKMARK [3][]{Outline0.3.1.10}{\376\377\127\372\116\216\137\072\123\026\133\146\116\140\166\204\147\000\117\030\215\357\137\204\211\304\122\022}{Outline0.3}% 7
\BOOKMARK [3][]{Outline0.3.2.13}{\376\377\131\032\145\340\116\272\147\072\123\117\124\014\214\003\136\246\173\227\154\325}{Outline0.3}% 8
\BOOKMARK [2][]{Outline0.4}{\376\377\116\377\167\037\176\323\147\234}{}% 9
